# curl -o /data/openpilot/selfdrive/dashcamd.py https://raw.githubusercontent.com/eFiniLan/openpilot-patches/master/0.5.12/ui_tweak/dashcamd.py && cd /data/openpilot && curl https://raw.githubusercontent.com/eFiniLan/openpilot-patches/master/0.5.12/ui_tweak/always_on_dashcam.diff | git apply -v
#
# This patch add a always on dashcam, it will record whatever on
# the screen when camera is on.
# add display_timestamp patch if you would like your video to have time
# on it.
#
# courtesy of pjlao307 (https://github.com/pjlao307/)
# this is just a always on version of his original implementation
# and integrated into openpilot service
#
===================================================================
--- a/selfdrive/manager.py
+++ b/selfdrive/manager.py
@@ -109,6 +109,7 @@
   "gpsd": ("selfdrive/sensord", ["./gpsd"]),
   "updated": "selfdrive.updated",
   "athena": "selfdrive.athena.athenad",
+  "dashcamd": "selfdrive.dashcamd",
 }
 android_packages = ("ai.comma.plus.offroad", "ai.comma.plus.frame")
 
@@ -147,6 +148,7 @@
   'proclogd',
   'ubloxd',
   'mapd',
+  'dashcamd'
 ]
 
 def register_managed_process(name, desc, car_started=False):
